### ISSUE TITLE
Back key navigation looks unnatural on mobile

### ISSUE THREAD

**Issue by xperiandri** [Odonno/modern-gitter-winjs#49]
I think that back key navigation is not compatible with hamburger menu.
Who agrees that he does not expect to go to previous page by pressing back key but navigate away to previous app?
It looks somewhat strange to navigate to some page while menu is available.

**Comment by Odonno** [comment]
@xperiandri I don't know. It looks normal to me. Do you have any example of a normal behavior ? (I usually copy the behavior of the Windows Store App)

**Comment by xperiandri** [comment]
Mail App does not use back key for navigation within an app

**Comment by Odonno** [comment]
You're right, the mail app does not handle back button. I'll see how to improve the navigation so. But one thing should remain : when you go back from a room, you should go back to the rooms list, no ?

**Comment by xperiandri** [comment]
No, because you have a button for room and a button for room list on a menu.
By the way you would better hide menu at all on mobile just keep hamburger button. Again like in Mail app. There is no need to take that place unnecessary.

Slack (Beta) also can be a good reference point.
Having rooms list right on the side bar could be convenient. You can switch almost immediately.
They have `+` button at the bottom of side bar which opens a new page to add an account. Here you can go back with a back key.
Your `Home` page is more like `About` and can also be moved to the bottom

And you can add quick room search like it is done in a new Store app. Where search button is placed opposite to hamburger button. And when you tap it reveals suggestion box at the top line hiding hamburger.
